contracts/core/tokens/RebalancingSetToken.sol

```diff
@@ -156,6 +152,12 @@ contract RebalancingSetToken is
             "RebalancingSetToken.constructor: Unit shares must be positive"
         );
 
+        // Require natural unit is non-zero
+        require(
+            _naturalUnit > 0,
```

Comment:
I think this should at least have a lower bound. If not an upper one too. I especially don't like the idea of a rebalancing set with a naturalUnit of 1. It will lead to a lot of failed rebalances I imagine when people try to just use unitShares = 1 on initialization.

Reply:
We could throw a minimumNaturalUnit and maximumNatural unit in the factory upon its construction. Maybe something between 10**4 and 10**12